index.bs

```diff
@@ -662,7 +662,7 @@ or {{DOMException}}
 should be used for errors,
 whether they are exceptions, promise rejection values, or properties.
 
-<h2 id="device-apis">Device APIs</h2>
+<h2 id="device-apis">Device and Wrapper APIs</h2>
```

Comment:
Maybe "OS Capability Wrapper APIs" instead? (As the overall section title? E.g., making it more generic as this advice recommends... get it?)